IBPSA/BoundaryConditions/Validation/UsersGuide.mo

```diff
@@ -0,0 +1,163 @@
+calculated using Horizontal radiation or dew point temperature and sky cover.
+</li>
+<li>Diffused radiation calculated using <a href=\"modelica://IBPSA.BoundaryConditions.SolarIrradiation.DiffusePerez\">Perez</a> and
+<a href=\"modelica://IBPSA.BoundaryConditions.SolarIrradiation.DiffuseIsotropic\">Isotropic</a> sky models</li>
+</ul>
+<h4>Outputs required</h4>
+<p><i>Annual Outputs</i></p>
+<p><b>&nbsp;</b>The following outputs are  provided for an annual simulation:</p>
+<ul>
+<li>Average dry bulb temperature (&deg;C)</li>
+<li>Average relative humidity (%)</li>
+<li>Average dewpoint temperature (&deg;C)</li>
+<li>Average humidity ratio (kg moisture/kg dry air)</li>
+<li>Average wet bulb temperature (&deg;C)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2)</li>
+</ul>
+<br><p><i>Hourly Outputs</i></p>
+<p>The following outputs are  provided for each hour of the days specified for each test case in Table 3:</p>
+<ul>
+<li>Dry bulb temperature (&deg;C)</li>
+<li>Relative humidity (%)</li>
+<li>Dewpoint temperature (&deg;C)</li>
+<li>Humidity ratio (kg moisture/kg dry air)</li>
+<li>Wet bulb temperature (&deg;C)</li>
+<li>Windspeed (m/s)</li>
+<li>Wind direction (degrees from north)</li>
+<li>Station pressure (mbar)</li>
+<li>Total cloud cover (tenths of sky)</li>
+<li>Opaque cloud cover (tenths of sky)</li>
+<li>Sky temperature (&deg;C)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2)&nbsp;</li>
+</ul>
+<br><p><i>Table 3: Specific Days for Output</i></p>
+<table summary = \"Specific Days for Output\" cellspacing=\"2\" cellpadding=\"0\" border=\"1\"><tr>
+<td><p>Case </p></td>
+<td><p>Days</p></td>
+</tr>
+<tr>
+<td><p>WD100 </p></td>
+<td><p>May 4th, July 14th, September 6th</p></td>
+</tr>
+<tr>
+<td><p>WD200 </p></td>
+<td><p>May 24th, August 26th</p></td>
+</tr>
+<tr>
+<td><p>WD300 </p></td>
+<td><p>February 7th, August 13th</p></td>
+</tr>
+<tr>
+<td><p>WD400 </p></td>
+<td><p>January 24th, July 1st</p></td>
+</tr>
+<tr>
+<td><p>WD500 </p></td>
+<td><p>March 1st, September 14th</p></td>
+</tr>
+<tr>
+<td><p>WD600 </p></td>
+<td><p>May 4th, July 14th, September 6th</p></td>
+</tr>
+</table>
+<br><p><i>Sub-hourly Outputs</i></p>
+<p>The following outputs are  provided at each timestep of the days specified for each test case in Table 3:</p>
+<ul>
+<li>Dry bulb temperature (C)</li>
+<li>Relative humidity (%)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2) </li>
+</ul>
+<p>The following outputs are  provided integrated hourly for the days specified for each test case in Table 3:</p>
+<ul>
+<li>Total incident horizontal solar radiation (Wh/m2)</li>
+<li>Total incident horizontal beam solar radiation (Wh/m2)</li>
+<li>Total incident horizontal diffuse solar radiation (Wh/m2) </li>
+</ul>
+<h4>Validation results</h4>
+<p>(Not available yet)</p>
```

Comment:
Why are the results not available yet?

Reply:
We are still working with the BESTEST team to adress some issues related to the diffuse radiation model.